src/gui/folderwizard.h

```diff
@@ -60,6 +60,16 @@ class FolderWizardLocalPath : public FormatWarningsWizardPage
     void cleanupPage() override;
 
     void setFolderMap(const Folder::Map &fm) { _folderMap = fm; }
+
+signals:
+    void styleChanged();
+
+public slots:
+    void slotStyleChanged();
```

Comment:
@claucambra I agree, since this slot is not connected to another class, you may as well make it private
`connect(this, &FolderWizardLocalPath::styleChanged, this, &FolderWizardLocalPath::slotStyleChanged);`

Having said that, I don't get why you need these signals/slots, as you may as well call the method via `void FolderWizardLocalPath::changeEvent(QEvent *e)` directly.

Reply:
Good point, now just have a private method called by changeEvent